Sources/SpeziHealthCharts/MetricChart/ChartViews/Modifiers/HealthChart+DisableInteractions.swift

```diff
@@ -0,0 +1,22 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+extension EnvironmentValues {
+    @Entry var disabledChartInteractions: HealthChartInteractions = []
+}
+
+
+extension HealthChart {
+    public func disable(interactions newValue: HealthChartInteractions) -> some View {
```

Comment:
I think having an extension on `View` would provide the most flexibility. Though, to not collide with the existing `disabled(_:)` modifier we might want to find a more unique name like `healthChartInteractions(disabled:)` maybe? Some of the accessibility modifiers have a similar naming scheme, where the external parameter name express the semantic effect of the modifier.